docs/initial-assessment.md

```diff
@@ -0,0 +1,23 @@
+# Initial self-assessment against draft goals
+
+*I'm team thoughtbot*
+* I contribute to thoughtbot being a good place to work (1/5 - this should just naturally grow over time)
+* I iterate over my existing knowledge effectively (2/5 - I have the basic learning how to learning skills)
+* I’m a great pair programmer (3/5 - Feedback from peers at Makers was positive)
+* I help others become better (1/5 - Again, this should grow over time)
+* I’m passionate about software/products (3/5)
+* I set my own direction (4/5)
+
+*I'm a developer*
+* I can write high quality code for React Native (OOP, easy to understand, easy to change) (2/5 - Know OOP basics and basic React)
+* I can write high quality test suites for React Native (1/5 - Only simple unit tests)
+* I can build apps in React Native (1.5/5 - could confidently build a frontend only app)
+* I understand how React Native works (2/5 - could explain the basics)
+* I understand how the web works (3/5)
+* I understand how databases work (2/5)
+
+*I'm a consultant*
+* I communicate clearly with clients (3/5 - I think my Product background should set me up well)
+* I can clearly explain to others where/when thoughtbot can add value (1/5)
+* I understand how technical/design decisions will impact clients (2.5/5 - PO experience)
+* My teams build software clients’ love (0.5/5 - Contributions will grow)
```

Comment:
@purinkle

Reply:
All looks good to me.

I'd merge this and add anything further in future PRs.

We should talk about what you want to work on first and how we are going to track this.